docs/it/authorization_endpoint.rst

```diff
@@ -183,54 +161,31 @@ Nel caso di utilizzo del parametro **scope** i seguenti valori sono supportati:
     - *email*,
     - *email_verified*.
 
-Gli attributi richiesti tramite il parametro **scope** sono disponibili sia nell'ID Token e sia nella risposta allo userinfo endpoint.
+.. admonition:: |cieid-icon|
+
+    Gli attributi richiesti tramite il parametro **scope** sono disponibili sia nell'ID Token e sia nella risposta allo *userinfo endpoint*.
+
+.. admonition:: |spid-icon|
+
+    Gli attributi richiesti tramite il parametro **scope** sono disponibili nella risposta allo *userinfo endpoint*.
 
 .. note::
-    Il parametro **scope** PUÒ contenere uno o più valori separati da uno spazio. Ad esempio l'utilizzo congiunto di *profile* e *email* permette di ottenere l'unione degli insiemi degli attributi (Minimum Dataset eIDAS e l'email).
 
-Nel caso di richiesta di singoli attributi dell'utente o specifiche combinazioni di essi, Il RP PUÒ usare il parametro **claims**. 
-Per la definizione del parametro **claims** e la modalità di utilizzo per la richiesta degli attributi dell'utente si può fare riferimento a `OpenID.Core#ClaimsParameter`_. 
+    Il parametro **scope** PUÒ contenere uno o più valori separati da uno spazio. Ad esempio l'utilizzo congiunto di *profile* e *email* permette di ottenere l'unione degli insiemi degli attributi (Minimum Dataset eIDAS e l'email).
+    Nel caso di richiesta di singoli attributi dell'utente o specifiche combinazioni di essi, Il RP PUÒ usare il parametro **claims**.
+    Per la definizione del parametro **claims** e la modalità di utilizzo per la richiesta degli attributi dell'utente si può fare riferimento a `OpenID.Core#ClaimsParameter`_.
 
 .. warning::
-    - Solo per CIE id: Nell'oggetto *id_token* del parametro **claims** è possibile richiedere solo il Minimum Dataset eIDAS. Gli altri attributi dell'utente DEVONO essere richiesti nell'oggetto *userinfo* del parametro **claims**. Inoltre, gli attributi utente richiesti nell'oggetto *id_token* sono disponibili anche allo *userinfo endpoint*. 
 
-.. warning::
-    - Se il parametro **claims** non è presente o non è valorizzato, viene restituito solo il claim *sub* nella risposta allo userinfo endpoint. 
+  Se il parametro **claims** non è presente o non è valorizzato, viene restituito solo il claim *sub* nella risposta allo *userinfo endpoint*.
 
-La tabella seguente mostra alcuni esempi di utilizzo.
+.. admonition:: |spid-icon|
 
```

Comment:
Solo per SPID:

Solo se il metadata del RP contiene id_token_encrypted_response_alg e id_token_encrypted_response_enc  ed il loro valore è diverso da 'none', gli attributi utente richiesti tramite l'oggetto id_token del parametro claims saranno disponibili nell'id_token restituito dall'endpoint token, in aggiunta agli altri claim eventualmente richiesti tramite il parametro scope.

Gli attributi utente richiesti tramite l'oggetto userinfo del parametro claims saranno disponibili solo nella risposta ottenuta dall'endpoint userinfo, in aggiunta agli altri claim eventualmente richiesti tramite il parametro scope.

@nunzionapoli gentilmente verifica se tale comportamento è conforme a OIDC Core.